### DISCUSSION TITLE
CefSharp or WebControl ?

### DISCUSSION THREAD

**Discussion by apovetkin** [unoplatform/uno#7925] [Q&A]
Hi guys,

Could you tell me if Uno Platform supports CefSharp Lib or any other WebControl?

Any example very appreciated.

Thanks,
Alexander

**Comment by jeromelaban** [comment]
Can you tell which target platform you are interested in ?

**Reply by apovetkin** [reply]
Windows, ANDROID and MacOS

**Reply by jeromelaban** [reply]
Running on Windows using UWP and WinUI 3 supports WebView2, as for Android and macOS, you can use the WebView control.

**Reply by apovetkin** [reply]
Thanks!
